docs/recovery.md

```diff
@@ -0,0 +1,351 @@
+ 1. ability   2. noise   3. lift     4. document
+ 5. certain   6. month   7. shoot    8. perfect
+ 9. matrix   10. mango  11. excess  12. turkey
+13. river    14. pitch  15. fluid   16. rack
+17. drill    18. text   19. buddy   20. pool
+21. soul     22. fatal  23. ship    24. jelly
+---------------END LND CIPHER SEED-----------------
+
+!!!YOU MUST WRITE DOWN THIS SEED TO BE ABLE TO RESTORE THE WALLET!!!
+```
+
+### Wallet+Seed Passphrases
+
+During the creation process, users are first prompted to enter a **wallet
+password**:
+```
+Input wallet password:
+Confirm wallet password:
+```
+
+This password is used to _encrypt_ the wallet on disk, which includes any
+derived master private keys or public key data.
+
+Users can also _optionally_ enter a second passphrase which we call the _cipher
+seed passphrase_:
+```
+Your cipher seed can optionally be encrypted.
+Input your passphrase if you wish to encrypt it (or press enter to proceed without a cipher seed passphrase):
+```
+
+If specified, then this will be used to encrypt the cipher seed itself. The
+cipher seed format is unique in that the 24-word phrase is actually a
+_ciphertext_. As a result, there's no standard word list as any arbitrary
+encoding can be used. If a passprhase is specified, then the cipher seed you
+write down is actually an _encryption_ of the entropy used to generate the BIP
+32 root for the wallet. As a result, it can even be left in plain site if one
+chooses as the passphrase is run through a standard KDF with modern parameters.
+Unlike a BIP 39 24-word phrase, the cipher seed is able to _detect_ incorrect
+passphrase. BIP 39 on the other hand, will instead silently decrypt to a new
+(likely empty) wallet.
+
+### Starting On-Chain Recovery
+
+The initial entry point to trigger recovery of on-chain funds in the command
+line is the `lncli create` command.
+```
+⛰   lncli create
+```
+
+Next, one can enter a _new_ wallet password to encrypt any newly derived seeds
+as a result of the recovery process.
+```
+Input wallet password:
+Confirm wallet password:
+```
+
+nce a new wallet password has been obtained, the user will be prompted for
+their _existing_ cipher seed:
+```
+Input your 24-word mnemonic separated by spaces: ability noise lift document certain month shoot perfect matrix mango excess turkey river pitch fluid rack drill text buddy pool soul fatal ship jelly
+```
+
+If a _cipher seed passphrase_ was used when the seed was created, it MUST be entered now:
+```
+Input your cipher seed passphrase (press enter if your seed doesn't have a passphrase):
+```
+
+Finally, the user has an option to choose a _recovery window_:
+```
+Input an optional address look-ahead used to scan for used keys (default 2500):
+```
+
+The recovery window is a metric that the on-chain rescanner will use to
+determine when all the "used" addresses have been found. If the recovery window
+is 2, then after we find to consecutive addresses that haven't yet been found,
```

Comment:
s/to/two

since the rescan doesn't actually exit on this condition, this sentence might be more accurate by saying "If the recovery window is 2, lnd will fail to find funds in any addresses generated after the point in which 2 consecutive addresses were generated but never used."